client/express-checkout/utils/index.ts

```diff
@@ -108,3 +110,68 @@ export const getErrorMessageFromNotice = ( notice: string ) => {
 	div.innerHTML = notice.trim();
 	return div.firstChild ? div.firstChild.textContent : '';
 };
+
+/**
+ * Returns the appearance settings for the Express Checkout buttons.
+ * Currently only configures border radius for the buttons.
+ */
+export const getExpressCheckoutButtonAppearance = () => {
+	return {
+		// variables: { borderRadius: '99999px' },
+	};
+};
+
+/**
+ * Returns the style settings for the Express Checkout buttons.
+ */
+export const getExpressCheckoutButtonStyleSettings = () => {
+	const buttonSettings = getExpressCheckoutData( 'button' );
+
+	const mapWooPaymentsThemeToButtonTheme = (
+		buttonType: string,
+		theme: string
+	) => {
+		switch ( theme ) {
+			case 'dark':
+				return 'black';
+			case 'light':
+				return 'white';
+			case 'light-outline':
+				if ( buttonType === 'googlePay' ) {
+					return 'white';
+				}
+
+				return 'white-outline';
+			default:
+				return 'black';
+		}
+	};
+
+	return {
+		paymentMethods: {
+			applePay: 'always',
+			googlePay: 'always',
+			link: 'auto',
+		},
+		layout: { overflow: 'never' },
+		buttonTheme: {
+			googlePay: mapWooPaymentsThemeToButtonTheme(
+				'googlePay',
+				buttonSettings?.theme ?? 'black'
+			),
+			applePay: mapWooPaymentsThemeToButtonTheme(
+				'applePay',
+				buttonSettings?.theme ?? 'black'
+			),
+		},
+		buttonType: {
+			googlePay: buttonSettings?.type ?? 'buy',
+			applePay: buttonSettings?.type ?? 'plain',
+		},
```

Comment:
When I select the call to action `only icon`, nothing is rendered. In this case the value of `buttonSettings.type` is `default`.

Reply:
``` 
Uncaught IntegrationError: Invalid value for elements.create('expressCheckout', options): buttonType.applePay should be one of the following strings: add-money, book, buy, check-out, contribute, donate, order, plain, reload, rent, subscribe, support, tip, top-up. You specified: default.
```

Reply:
Nice catch, this should be a _relatively_ simple fix. Thank you!

Reply:
Fixed in 3a3dd6c0722b408085a3bd5055617abcb0cf54d6 🙌